tests/suites/os/tests/device-tree/index.js

```diff
@@ -213,7 +214,31 @@ module.exports = {
 					targetState.local.config.HOST_CONFIG_dtparam,
 					'DTparams successfully configured in config.txt',
 				);
+				/* Static binary currently shared by RPI maintainer in gdrive only
+				 * See: https://github.com/raspberrypi/Raspberry-Pi-OS-64bit/issues/67#issuecomment-653209729
+				 */
+				test.is(
+					await this.context
+						.get()
+						.worker.executeCommandInHostOS(
+							'cd /tmp/ && curl -L "https://drive.google.com/uc?export=download&id=1HS9E5vnxxNqrizB4mEYrnFoQQ1axSRKm" -o vcdbg && chmod +x ./vcdbg && \
+							./vcdbg log msg 2>&1 | grep -q -i "File read:" ; echo $?',
+							this.context.get().link,
+						),
+						'0',
+						'vcdbg static binary should be downloaded and run successfuly'
+				);
+				test.is(
+					await this.context
+						.get()
+						.worker.executeCommandInHostOS(
+							'cd /tmp/ && ./vcdbg log msg 2>&1 | grep -q -i "Failed to load" ; echo $?',
```

Comment:
@acostach @klutchell 

I think we are only checking if no dtoverlays were not loaded successfully.

Do we have a test case where we set different dtoverlays?
e.g `dtoverlay=vc4-kms-v3d` and `dtoverlay=vc4-fkms-v3d` would be two different scenarios

Reply:
@rahul-thakoor right, this checks the overlay that comes in the config.txt in the image, be it either fkms or kms. We can add the both.